ale_linters/python/mypy.vim

```diff
@@ -39,14 +71,21 @@ function! ale_linters#python#mypy#GetCommand(buffer) abort
     let l:dir = s:GetDir(a:buffer)
     let l:executable = ale_linters#python#mypy#GetExecutable(a:buffer)
 
+    let l:daemon_enabled = s:DaemonEnabled(a:buffer)
```

Comment:
This isn't totally ideal because `GetExecutable` is already potentially going to do an upsearch looking for `.dmypy.json`. Better would be to call that once, pass it to `GetExecutable(a:buffer)` and reuse it here.

I'm not sure if this is important, but pointing it out.